src/Tizen.NUI/src/public/BaseComponents/View.cs

```diff
@@ -429,6 +429,8 @@ static View()
                 DispatchHoverMotionProperty = BindableProperty.Create(nameof(DispatchHoverMotion), typeof(bool), typeof(View), false,
                     propertyChanged: SetInternalDispatchHoverMotionProperty, defaultValueCreator: GetInternalDispatchHoverMotionProperty);
 
+                OffScreenRenderingProperty = BindableProperty.Create(nameof(OffScreenRendering), typeof(OffScreenRenderingType), typeof(View), default(OffScreenRenderingType),
```

Comment:
I don't care about it but.. should we support it at xaml case also?

Reply:
https://github.com/Samsung/TizenFX/pull/6345/files#diff-c02892a5791f4c2f47c661706b561a736af7f6b408e3f435811e8d136f91dc19R1292-R1317

At this PR, I didn't add BindableProperty, since we decide to do not support xaml case what we newly, created.

Please check it, and call getter/setter to Native directly, instead of BindableProperty.